examples/llama2/llama2_qlora.json

```diff
@@ -2,9 +2,17 @@
     "input_model": {
         "type": "PyTorchModel",
         "config": {
-            "model_script": "user_script.py",
-            "io_config": "get_merged_decoder_with_past_io_config",
-            "dummy_inputs_func": "get_merged_decoder_with_past_dummy_inputs",
+            "io_config": {
+                "input_names": ["input_ids", "attention_mask"],
+                "output_names": ["logits"],
+                "input_shapes": [[2,8], [2,40]],
+                "input_types": ["int32", "int32"],
+                "dynamic_axes": {
+                    "input_ids": {"0": "batch_size", "1": "sequence_length"},
+                    "attention_mask": {"0": "batch_size", "1": "sequence_length"}
+                },
+                "kv_cache_config": true
```

Comment:
>kv_cache_config [](http://example.com/codeflow?start=17&length=15)

better have a way to specify the kv_cache if user would like to use io_config function.

Reply:
I think user can directly implement the kv_cache in their function. The way of io_config function still exists and works.

Reply:
So, it seems user would not be allowed to use the kv_cache config if he write function? what's your thought? @devang-ml

Reply:
Based our offline discussion,if user is writing io_config user function then it is fair to expect them to write their own kv_cache inputs in their scripts.